### PR TITLE
Feature/improve load time

### PR DESCRIPTION
**Pull request by jonathan016**
The currently deployed version (per 27 Jan 2020) loads very slowly.

Attempted to improve load time by:
- Chunking with Webpack
- Async import pages in router
- Enabling gzip compression with Webpack's `CompressionPlugin`

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -52,6 +60,57 @@ module.exports = {
...
+              enforce: true
+            },
+            fortawesome: {
+              test: /[\\/]node_modules[\\/]@fortawesome(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'fortawesome',
+              enforce: true
+            },
+            mavon_editor: {
+              test: /[\\/]node_modules[\\/]mavon-editor(.[a-zA-Z0-9.\-_]+)[\\/]/,
```
Comment (oliverrsebastian) [inline vue.config.js]: i thought that this will be deleted later? if left there, there won't be any problem right?

File: vue.config.js [hunk 2]
```diff
@@ -52,6 +60,57 @@ module.exports = {
...
+              enforce: true
+            },
+            vue_moment: {
+              test: /[\\/]node_modules[\\/]vue-moment(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'vue-moment',
+              enforce: true
+            },
+            vue_toasted: {
+              test: /[\\/]node_modules[\\/]vue-toasted(.[a-zA-Z0-9.\-_]+)[\\/]/,
```
Comment (oliverrsebastian) [inline vue.config.js]: this as well

File: vue.config.js (hunk 1 above)
Comment (jonathan016) [reply vue.config.js]: No problem, if it is removed then this section can be removed as well. Even if this section is not removed, the result would be no `mavon-editor.js` chunk. @oliverrsebastian

File: vue.config.js (hunk 2 above)
Comment (jonathan016) [reply vue.config.js]: See https://github.com/function-future/function-frontend/pull/239#discussion_r371592813, with difference the result would be no `vue-toasted.js` chunk. @oliverrsebastian

**Pull request merged**